### ISSUE TITLE
tuigreet doesn't respect the "vt.global_cursor_default=0" boot paramater

### ISSUE THREAD

**Issue by hossamdash** [apognu/tuigreet#56]
one of the steps to achieve [silent boot](https://wiki.archlinux.org/title/Silent_boot) is to append the "vt.global_cursor_default=0" boot parameter to your boot loader config. tuigreet seems to show the blinking cursor after pressing enter on login and on logout. It should respect this parameter.

**Comment by apognu** [comment]
`tuigreet` exits on successful login and is not yet started again on logout (until the UI is shown again). I need to check the exact timeline of events, though. Are you sure the cursor you're seeing is coming from `tuigreet` and not `greetd` or your VT?

As I understand it, this kernel parameter is for VTs to interpret, not applications.

**Comment by hossamdash** [comment]
I can confirm it's not coming from my VT as I when i use [ly](https://github.com/nullgemm/ly) it seems to respect setting this parameter. Also normal VTs without any display manager or greetd in them don't have any blinking cursors. I'm not sure how I will check whether it's coming from greetd or tuigreet. I will try using greetd with agetty to check maybe?

**Comment by hossamdash** [comment]
no blinking cursor with greetd using agreetty. I was able to achieve silent boot with the following config and can confirm there were no blinking cursors after login or shutdown. tuigreet displays these blinking cursors
![image](https://user-images.githubusercontent.com/61992392/152939889-402a74ac-262f-4f05-b558-e84d20171abb.png)

**Comment by apognu** [comment]
I configured my system the same way, and I can't say the behavior of `agreety` matches what you described:
 * It still shows the cursor at the prompt (where native `tty` prompts do not).
 * It does show a cursor after it exits, until `greetd` actually starts the session.

I also tried removing all cursor interaction from `tuigreet`, and the cursor indeed does not show at the prompt but it still shown in the corner after it exists until `sway` is started up.

Even when explicitely instructing `tuigreet` to hide the cursor before it exits (which I actually might do), it still reappears later on before the compositor is run.
